sqlmesh/core/plan/definition.py

```diff
@@ -440,6 +440,16 @@ def _missing_intervals(self) -> t.Dict[t.Tuple[str, str], Intervals]:
 
         return self.__missing_intervals
 
+    @property
+    def has_unmodified_unpromoted(self) -> bool:
+        """Is the plan for an existing dev environment, has the include unmodified flag, and contains unmodified models that have not been promoted."""
+        return (
+            self.is_dev
+            and not self.context_diff.is_new_environment
+            and self.include_unmodified
+            and len(self.context_diff.unpromoted_models) > 0
```

Comment:
and self.context_diff.unpromoted_models

Reply:
mypy was giving a type error when it was passed like that: 

```sqlmesh/core/plan/definition.py:447: error: Incompatible return value type (got "Union[Literal[False], Set[str]]", expected "bool")  [return-value]```

should i change the return type, type ignore, or something else?

Reply:
bool()